app/styles/headers/_fourth.scss

```diff
@@ -0,0 +1,65 @@
+.header__fourth {
+    max-width: 1180px;
+    width: 100%;
+    height: 100vh;
+    color: #fff;
+    font-family: $font;
+    margin: 0 auto;
+    line-height: 1.5;
+    background: #262626 url("../img/iPhone-bg.png") no-repeat center;
+        &__wrapp {
+            width: 80%;
+            height: 80vh;
```

Comment:
please, refactor height using paddings, because on huge display it will be probably broken down